rules/windows/defense_evasion_masquerading_communication_apps.toml

```diff
@@ -45,7 +43,7 @@ process where host.os.type == "windows" and
 
     /* Teams */
     (process.name : "Teams.exe" and not
-      (process.code_signature.subject_name == "Microsoft Corporation" and process.code_signature.trusted == true)
+      (process.code_signature.subject_name in ("Microsoft Corporation", "Microsoft Corporation") and process.code_signature.trusted == true)
```

Comment:
What's the difference between these 2 values?

Reply:
++

Reply:
Nice catch, there is no difference 😅